Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -182,6 +184,25 @@ def severity_to_level(severity):
         return 1
 
 
+def parse_notable(notable, to_dict=False):
+    """ Parses the notable
+    Args:
+        notable: The notable
+        to_dict: Whether to cast the notable to dict or not.
+    Returns: The parsed notable
+    """
+    notable = replace_keys(notable) if REPLACE_FLAG else notable
+    for key, val in notable.items():
+        # if notable event raw fields were sent in double quotes (e.g. "DNS Destination") and the field does not exist
+        # in the event, then splunk returns the field with the key as value (e.g. ("DNS Destination", "DNS Destination")
+        # so we go over the fields, and check if the key equals the value and set the value to be empty string
+        if key == val:
+            demisto.info('Found notable event raw field [{}] with key that equals the value - replacing the value '
+                         'with empty string'.format(key))
+            notable[key] = ''
+    return dict(notable) if to_dict else notable
```

Comment:
not sure i understand this, isnt `notable` of type `dict` already?

Reply:
Nope, it is of type `OrderedDict` and Demisto doesn't know how to handle it, so casting is needed in that case.

Reply:
please mention in the docstring that `notable` is of type `OrderedDict`